app/controllers/qa_controller.rb

```diff
@@ -89,6 +98,11 @@ def facet_grouped_allowed_values
   end
 
   def options
+    if single_custom_options_question?
```

Comment:
Can you add a test for this new option? Perhaps something similar to: https://github.com/alphagov/finder-frontend/blob/master/features/qa.feature?

Also, what happens if the question is a `single_custom_options` question, but there aren't any custom options set?